static/app/views/monitors/components/overviewTimeline/timelineTableRow.tsx

```diff
@@ -92,7 +95,9 @@ export function TimelineTableRow({
       ? [
           (env: string, isMuted: boolean) => ({
             label: isMuted ? t('Unmute Environment') : t('Mute Environment'),
```

Comment:
Let's have this check `monitor.isMuted` as well so we're not showing a "Unmute Environment` that can't be clicked